_posts/2019-11-27-monthly-sliding-window-partitioning-dhkeeping.md

```diff
@@ -0,0 +1,239 @@
+---
+layout: post
+title: Design decisions and problems faced with during database housekeeping operations
```

Comment:
Why is the title same as the other one? Can you at least make it part 1 / part 2 if you have to keep the title the same? If that's the route you're gonna go I would also check to see if there is some kind of special support for multi part posts in jekyll/git pages.